docs/source/video_tutorials.rst

```diff
@@ -0,0 +1,39 @@
+Video Tutorials
+===============
+
+Basics of *stk*
+---------------
```

Comment:
Overview of the recent publication covering the capability of _stk_ and the docs of _stk_. 

Jupyter Notebook is available at https://github.com/andrewtarzia/stk-examples/tree/main/notebooks_from_videos .